### PR TITLE
[8] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by Olawumi11**
## Task: `Add page navigation to your pages`
- `created a sign up page`
-'added page navigation to my pages'

### CODE REVIEW

File: submissions/famuyiwaOlawumi/html/htmlform-7.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+</head>
+<body>
+    <p>
+        <a href="../html-forms/htmlform-8.html">Signup</a>
+        <a href="../htmltask/index.html">Product page</a>
+        <a href="htmlform-7.html">Contact Us</a>
+    </p>
+    <h1>Contact Form</h1>
+<form action="#" method="post" enctype="multipart/form-data">
+    <fieldset>
```
Comment (adaugochi) [inline submissions/famuyiwaOlawumi/html/htmlform-7.html:15]: Please what page is this

File: submissions/famuyiwaOlawumi/html-forms/htmlform-8.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>signup page</title>
+</head>
+<body>
+    <p>
+        <a href="htmlform-8.html">Signup</a>
+        <a href="../htmltask/index.html">Product page</a>
```
Comment (adaugochi) [inline submissions/famuyiwaOlawumi/html-forms/htmlform-8.html:10]: navigation links is for task 8. Please remove them

Comment (Olawumi11) [reply submissions/famuyiwaOlawumi/html-forms/htmlform-8.html:10]: This is particular pull request for task 8 as the name implies ma'am. The instruction was to create pull request for each task. Nothing was said about waiting for one to be merged before creating another.
Thanks

File: submissions/famuyiwaOlawumi/html/htmlform-7.html (hunk 1 above)
Comment (Olawumi11) [reply submissions/famuyiwaOlawumi/html/htmlform-7.html]: Sign up page

**Comment by Olawumi11** [comment]
I didn't close any of my pull request, the first pull request was for task [7] contact form

**Pull request closed without merging**